docs/manual/grelfunctions.md

```diff
@@ -447,6 +447,10 @@ As of OpenRefine 3.4.1, uniques() reorders the array items it returns; in 3.4 be
 
 ## Date functions {#date-functions}
 
+:::info
+Date/times without timezone info were interpreted as **local** up until May 2018 when OpenRefine 3.0 was released, at which point they were switched from **local** to **UTC**.  One benefit of this change was to introduce consistency and reproducibility when working collaboratively and sharing projects across timezones.  Discussed in issue [#6009](https://github.com/OpenRefine/OpenRefine/issues/6009)
```

Comment:
The most relevant place for this information is with the `toDate()` and `toString()` documentation, although those should probably be brief references to a more general discussion.

Reply:
I wanted to have this generally at the beginning of discussing all the Date functions.
But would love your help on the `toString()` documentation suggestions which I can add in this PR.